Alignment/OfflineValidation/python/TkAlAllInOneTool/configTemplates.py

```diff
@@ -17,6 +17,16 @@
 
 ######################################################################
 ######################################################################
+loadGlobalTagTemplate="""
+#Global tag
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag,".oO[GlobalTag]Oo.")
```

Comment:
@ghellwig my understanding from looking at the code was that this will work both for the auto: global tags and for the real ones (and actually we could also specify other conditions through this interface).  Would you be able to confirm?

Thanks!